src/error.js

```diff
@@ -361,7 +368,14 @@ export function reportErrorToServerOrViewer(win, data) {
   return maybeReportErrorToViewer(win, data).then((reportedErrorToViewer) => {
     if (!reportedErrorToViewer) {
       const xhr = new XMLHttpRequest();
-      xhr.open('POST', urls.errorReporting, true);
+      // Override the errorReportingUrl to test the new error reporting endpoint.
+      const newErrorReportingUrl =
+        'https://us-central1-amp-error-reporting.cloudfunctions.net/r';
```

Comment:
Note: we need to add a `errorReportingBeta` field to URLs. If done, we could set the beta field to this URL. When we fully migrate, we can update `errorReporting` field to this URL and update `errorReportingBeta` to the new beta URL.

Reply:
Introduced this field in #27591